tests/unit/pre_processing/test_pre_processing.py

```diff
@@ -91,6 +91,7 @@ def test_different_stage_transforms() -> None:
         assert isinstance(processed_batch, torch.Tensor)
         assert processed_batch.shape == (1, 3, 288, 288)
 
+    @pytest.skip
```

Comment:
why are these skipped?

Reply:
They are not needed anymore. These tests check the setting of the transforms in the datamodule and dataloader from the preprocessor, which we don't do anymore. I skipped them for now, but once we settle on the final design they could be removed.

Reply:
ah right, I see